src/parallel/pipeline.cpp

```diff
@@ -167,6 +176,8 @@ void Pipeline::Schedule() {
 		}
 		break;
 	}
+	case PhysicalOperatorType::CREATE:
+	case PhysicalOperatorType::UNNEST:
```

Comment:
`UNNEST` is not a sink, so it doesn't need to be here (this will never be `UNNEST`).